src/components/Match/matchColumns.jsx

```diff
@@ -644,6 +644,13 @@ export const analysisColumns = [heroTdColumn, {
   ),
 }];
 
+const playerDeaths = (row, col, field) => (
+  field > 0 && <div className={styles.playerDeath}>
```

Comment:
Feels duplicated to have the markup twice.  Could we render just the number of skulls that `field` indicates?  (In practice it should always be 0, 1, or 2, although 3 is technically possible with aegis/WK ult since those get counted as deaths)